VideoLocker/src/main/java/org/edx/mobile/view/adapters/InfiniteScrollUtils.java

```diff
@@ -124,14 +125,32 @@ public void loadMore() {
         }
 
         private void onLoadMore() {
+            onLoadMore(true, false);
+        }
+
+        /**
+         * Variant of the {@link #onLoadMore} function allowing us to control the visibility of
+         * progress and lazily clear adapter after the page has loaded.
+         *
+         * @param showProgress         <code>true</code> if we don't want to show progress while
+         *                             loading, otherwise <code>false</code>.
+         * @param lazyClearanceEnabled <code>true</code> if we want to clear after the page has
+         *                             loaded, otherwise <code>false</code>.
+         */
+        private void onLoadMore(boolean showProgress, final boolean lazyClearanceEnabled) {
```

Comment:
Since the two parameters always have the opposite value, it might make sense to condense them to just one here.